ipyevents/events.py

```diff
@@ -13,6 +13,17 @@ class Event(CoreWidget):
     watched_events = List().tag(sync=True)
     ignore_modifier_key_events = Bool(False).tag(sync=True)
     prevent_default_action = Bool(False).tag(sync=True)
+    xy_coordinate_system = Unicode(allow_none=True, default=None).tag(sync=True)
+    _xy_coordinate_system_allowed = List([
+        None,       # Not tracking mouse x/y
+        'array',    # "natural" coordinates for the widget (e.g. image)
+        'client',   # Relative to the visible part of the web page
+        'offset',   # Relative to the padding edge of widget
+        'page',     # Relative to the whole document
+        'relative', # Relative to the widget
+        'screen'    # Relative to the screen
+    ]).tag(sync=False)
```

Comment:
This doesn't even need to be a traitlet -- you could just make it a set: 
```
_xy_coordinate_system_allowed = {
    None, 'array', ...
}
```
It doesn't matter that much, but the fact that it's a traitlet implies that you want to somehow react to user changes on it, so it's a bit confusing for the reader.

Reply:
good point!